### ISSUE TITLE
Access git info for code coverage tools

### ISSUE THREAD

**Issue by cavis** [PRX/Infrastructure#33]
Tools such as Codecov normally pull data from the `.git` directory... the repo name, sha, branch, etc.  Seems that `.git` is [not present when going codepipeline -> codebuild](https://forums.aws.amazon.com/thread.jspa?messageID=754120&tstart=0).

We need some way to access this data.  Either get the `.git` folder somehow, or figure out if the sha/branch/etc can be passed directly to the `codecov` command.

**Comment by farski** [comment]
@cavis Shouldn't Codecov just run against the repository? What is the benefit to having it run in CodeBuild as well?

**Comment by cavis** [comment]
Rubocop (and other static analyzers) can run against the static codebase.  Coverage tools have to be enabled during the test run.  And you give them access to `.git` so they can tell how well your branch/PR has covered the files you touched.

**Comment by cavis** [comment]
Yeah, what I've done in Dovetail and other repos works (passing in a whole slew of GIT-env variables).  But it is a bit inconvenient, and requires a custom fork of CodeCov.  So we should probably still figure this out at some point, for non-js repos.

**Comment by farski** [comment]
https://aws.amazon.com/about-aws/whats-new/2020/09/aws-codepipeline-now-supports-git-clone-for-source-actions/

**Comment by farski** [comment]
https://github.com/PRX/Infrastructure/commit/acbcc8d987d4b0978ca87a881f4468d6756f638a
